src/thriveopedia/ThriveopediaPage.cs

```diff
@@ -29,6 +29,8 @@ public abstract class ThriveopediaPage : PanelContainer
     /// </summary>
     public abstract string TranslatedPageName { get; }
 
+    public abstract string? ParentPageName { get; }
```

Comment:
Could you add a comment on this what this stores / is? I can't immediately tell from the name of this property what this is for.

Reply:
Added.